api-examples/class.AsyncMysqlClient/adoptConnection/001-basic-usage.php.typechecker.expect

```diff
@@ -1 +0,0 @@
-No errors!
```

Comment:
Shouldn't this file be deleted entirely? Or am I misunderstanding the GitHub UI here?

Reply:
I verified they're deleted. It says "This file was deleted." in the GitHub UI for me, but that message disappears when you expand the file so it's pretty confusing.